snapcraft/cli/_errors.py

```diff
@@ -86,16 +94,18 @@ def exception_handler(  # noqa: C901
         - a snapcraft handled error occurs, debug=False so only the
           exception message is shown
     """
-    # TODO pickle the traceback if on a manged host so the actual host can deal with it.
     exc_info = (exception_type, exception, exception_traceback)
     exit_code = 1
-    is_snapcraft_error = issubclass(exception_type, errors.SnapcraftError)
+    is_snapcraft_error = (
+        issubclass(exception_type, errors.SnapcraftError)
+        and exception_type is not ProviderExecError
```

Comment:
Ah, this is where the magic happens. We do however lose the traceback information to what led to this exception from sentry's point of view though, right?